drivers.xml

```diff
@@ -654,6 +654,10 @@
 		<device label="DeepSkyDad FR1" manufacturer="DeepSkyDad">
             <driver name="DeepSkyDad FR1">indi_deepskydad_fr1</driver>
             <version>1.0</version>
+        </device>
+                 <device label="Wanderer Rotator Lite" manufacturer="Wanderer Astro">
+            <driver name="Wanderer Rotator Lite">indi_WandererRotatorLite</driver>
```

Comment:
As the convention, the executable should perhaps be

indi_wandererlite_rotator

Reply:
Fixed,thanks :)